src/frameworks/react/index.js

```diff
@@ -1,3 +1,167 @@
+const fse = require('fs-extra');
+const inquirer = require('inquirer');
+const path = require('path');
+
+/* eslint-disable no-useless-escape */
+const packageJsonScripts = {
+  scripts: {
+    // 'build:watch': 'node scripts/start.js',
+    // dev: 'concurrently \"npm run build:watch\" \"env-cmd npm start\"',
+    dev: 'react-scripts start',
+    start: 'node server/index.js',
+    stylelint: 'stylelint \"src/assets/scss/**/*.scss\"',
+    lint: 'eslint . && npm run stylelint',
+  },
+};
+/* eslint-enable */
+
+const dependencies = {
+  dependencies: {
+    express: '^4.17.1',
+    'serve-static': '^1.14.1',
+  },
+  devDependencies: {
+    '@reduxjs/toolkit': '^1.6.0',
+    'env-cmd': '^10.1.0',
+    'eslint-config-airbnb-base': '^14.0.0',
+    'node-sass': '^5.0.0',
+    'react-redux': '^7.2.4',
+    'react-router-dom': '^5.2.0',
+    redux: '^4.1.0',
+    'redux-devtools': '^3.7.0',
+    stylelint: '^11.1.1',
+    'stylelint-config-recommended': '^3.0.0',
+    'stylelint-config-recommended-scss': '^4.0.0',
+    'stylelint-config-standard': '^19.0.0',
+    'stylelint-scss': '^3.12.1',
+    scss: '^0.2.4',
+  },
+};
+
 module.exports = async function react() {
-  this.log('todo');
+  const { projectName } = await inquirer.prompt({
+    type: 'input',
+    name: 'projectName',
+    message: 'Enter the folder name for your project',
+  });
+
+  await this.asyncCommand('npx', ['create-react-app', projectName]);
```

Comment:
Not sure if this is down to my environment or how the starter kit + CRA works, but I had to cd into the project dir and run an install again even after this command completes. There's a few things to note about this. First of all, if the end user of the Starter Kit has Yarn installed - like me - CRA will use that to install the dependencies, but everything else we set up uses NPM. So, we should add the flag for CRA to use NPM to install things, details can be found here: https://create-react-app.dev/docs/getting-started#selecting-a-package-manager The other thing, is once CRA runs, it leaves the user with a prompt to `cd` into the project dir, then run dev. This is why I was approaching this from a CRA template perspective so that we didn't have to do a second install after CRA ran. Hopefully, just adding the flag will eliminate this double-install and everything will be peachy.

Reply:
This may not be as clean as creating a CRA template, but I wanted to give it a shot. I'm wary of fully going the CRA template way because it's yet another thing that could block someone from feeling comfortable getting involved in the project.

Reply:
I did add the --use-npm flag. Hopefully that clears up any confusion that might cause.